.gitignore

```diff
@@ -297,6 +297,12 @@ fabric.properties
 
 # Rando
 creds.env
+development/*.txt
 
 # Invoke overrides
 invoke.yml
+
+# Docs
+docs/README.md
+docs/CHANGELOG.md
```

Comment:
Why are these gitignored?

Reply:
Not sure, it's from the template: https://github.com/networktocode-llc/cookiecutter-ntc/blob/nautobot-plugin-v1.0/files/.gitignore

Should I remove these lines?

Reply:
Harmless, those files were copied previously I think for some docs setup. We do not have these files in `docs` currently.

Reply:
Will remove these from the template, keeping it here for now.